src/Prism.Core/Commands/DelegateCommand.cs

```diff
@@ -46,7 +47,17 @@ public DelegateCommand(Action executeMethod, Func<bool> canExecuteMethod)
         ///</summary>
         public void Execute()
         {
-            _executeMethod();
+            try
+            {
+                _executeMethod();
```

Comment:
should we also add a try/catch around the _canExecuteMethod on line 69?

Reply:
done